### DISCUSSION TITLE
Pandora Behaviour Engine v1.0.0-alpha

### DISCUSSION THREAD

**Discussion by Monitor221hz** [Monitor221hz/Pandora-Behaviour-Engine-Plus#63] [Announcements]
# Changes
Fixed #62 (cause of #40, #25, #55, #52)
Fixed #60 which fixes #58

# IMPORTANT:
For the first time since initial release, this update REQUIRES users to delete the "meshes" folder in any old **Pandora output** before launching this version. (THIS DOES NOT MEAN THE MESHES FOLDER IN DATA). You only have to do this for the first successful run of this new version of the engine. 

**REINSTALL ANY CUSTOM SKELETON MODS YOU HAD.** Previous versions of Pandora mistakingly deleted any loose skeleton.hkx files because it just assumed they were old output, and I apologize for that. This includes mods like Baka ABC and Humanoid Vampire lord Bats power fix

<hr /><em>This discussion was created from the release <a href='https://github.com/Monitor144hz/Pandora-Behaviour-Engine-Plus/releases/tag/v1.0.0-alpha'>Pandora Behaviour Engine v1.0.0-alpha</a>.</em>

**Comment by Monitor221hz** [comment]
I'm taking a break for about a week or so. Development has drained all my free time and energy, if I don't take this break there will be major burnout down the line.

**Reply by DKLYNTLY** [reply]
you earned it

**Reply by DKLYNTLY** [reply]
it works no more deleting the anims in the character assets folder

**Reply by lobotomy-x** [reply]
The progress in the last month or so has been absolutely insane. I'm amazed you've kept going with such frequent updates. Enjoy the rest, you've earned it!

**Reply by MisterWolfe034** [reply]
Is the XPMSE patch needed in order to create a FNIS.esp for the load order or this should be generated by Pandora 1.0 by itlself?

**Comment by MK99MA** [comment]
First of all, thanks for your work and enjoy your well earned break from it.

Is there a known partial incompatibility with Immersive Interactions - Animated Actions (on "SE" so 1.5.97)? To be precise everthing seems to be working except for the 'Open Door' Animation (perhaps there are others too, but for this one it's rather obvious when its missing / not working).

Maybe this was already mentioned somewhere / I simply did something wrong (MO2 mod order when generating?) but then I'm either blind or didn't look properly.

**Reply by MK99MA** [reply]
Just wanted to update, it seems to affect every animation of the mod that isn't added by it (knocking, picking up, opening door animations).

**Comment by AntimonyF5** [comment]
Thanks for your contribution！ But the creature animations in SEXLAB still don't seem to work

**Reply by AntimonyF5** [reply]
Also, for some reason I can't block on the move

**Reply by AntimonyF5** [reply]
And it doesn't support for tktk‘s TKdodge and Ultimate Combat

**Comment by Kyationa** [comment]
hey do we need a patch for "poser hotkeys plus se"?

**Comment by Politom** [comment]
The crossbow animation is broken, in addition to other animations from other mods.
